compose/backend.yml

```diff
@@ -161,13 +153,15 @@ services:
       interval: 10s
       timeout: 10s
       retries: 10
-    environment:
-      DB_USERNAME:
-      DB_PASSWORD:
     ports:
-      - 5000:5432
+      - 5050:5432
```

Comment:
We would need to update the DB ports locally with this change, was this port change inevitable?

Reply:
Exactly, I had to do it, for example, in my local.
I struggled to find the port configuration for this, but really couldn't find it concluding that it is coming from inside the code or something (I wouldn't normally expect).
I decided that we follow the default, the reason being the SIDAM is a common app suite all other teams will also use. It would be worse to change a port of a common service rather than that of our own db.

Reply:
no worries then, probably worth mentioning the DB port change in the README

Reply:
Thank you Giri. I reflected this to readme, and actually this last suggestion reminded me of other improvements in readme.